src/qt/optionsmodel.cpp

```diff
@@ -98,7 +104,17 @@ void OptionsModel::Init(bool resetSettings)
 
     // These are shared with the core or have a command-line parameter
     // and we want command-line parameters to overwrite the GUI settings.
-    //
+    for (OptionID option : {DatabaseCache}) {
+        std::string setting = SettingName(option);
+        if (node().isSettingIgnored(setting)) addOverriddenOption("-" + setting);
+        try {
+            getOption(option);
+        } catch (const std::runtime_error& e) {
+            error = strprintf(_("Could not read setting \"%s\", %s."), setting, e.what());
```

Comment:
68a4f322e93e413cb8f1f1b998a2b2ee63df9b9d: how can you trigger this error?

Reply:
The only way to throw from `getOption()` is if `SettingName()` (called from `getOption()`) throws. However, `SettingName(option)` (same argument) is also called just before this `try/catch` without `try/catch`. So, if `SettingName()` is going to throw it will do so before this `try/catch` and so this code will never be reached.

To trigger that `SettingName()` to throw the source code must be modified. Cannot be triggered by wrong user input or corrupted config file.